tests/integration/cli/commands/test_issue_2050.py

```diff
@@ -101,7 +101,7 @@ def test_lock_uncompilable_sdist(
 
 
                 setup_kwargs = dict(
-                    name="pex.tests.bad-c-extension",
+                    name="pex_tests_bad_c_extension",
```

Comment:
This was the easiest way to deal with changes in how Pip, vendored setuptools and vendored packaging handle name normalization: just use a pre-normalized name, which skates through all combos of the above as a noop.